src/Components/SingleMovie/SingleMovie.js

```diff
@@ -13,72 +14,94 @@ class SingleMovie extends Component {
-              </h3>
-            </div>
-            <p className="movie-rating">
-              ⭐️ {Math.round(average_rating * 10) / 10}
-            </p>
-            <p className="tagline">{tagline}</p>
-            <p className="overview">{overview}</p>
-            <p className="budget-revenue">
-              Budget: {formatter.format(budget)} • Revenue:{' '}
-              {formatter.format(revenue)}
-            </p>
-            <div className="genre-container">
-              {genres.map((genre) => (
-                <p className="movie-genre">{genre}</p>
-              ))}
-            </div>
-          </div>
-        </section>
+        {this.state.singleMovie && (
+          <fragment>
+            <Header bannerImage={this.state.singleMovie.backdrop_path} />
+            <section className="single-movie-container">
+              <div className="poster-rating-styling">
+                <img
+                  className="single-movie-poster"
+                  src={this.state.singleMovie.poster_path}
+                  alt={this.state.singleMovie.title}
+                />
+              </div>
+              <div className="movie-details">
+                <div className="movie-title-styling">
+                  <h2>{this.state.singleMovie.title}</h2>
+                  <h3>
+                    {this.state.singleMovie.release_date} •{' '}
+                    {this.state.singleMovie.runtime}mins
+                  </h3>
+                </div>
+                <p className="movie-rating">
+                  ⭐️{' '}
+                  {Math.round(this.state.singleMovie.average_rating * 10) / 10}
+                </p>
+                <p className="tagline">{this.state.singleMovie.tagline}</p>
+                <p className="overview">{this.state.singleMovie.overview}</p>
+                <p className="budget-revenue">
+                  Budget: {formatter.format(this.state.singleMovie.budget)} •
+                  Revenue: {formatter.format(this.state.singleMovie.revenue)}
+                </p>
+                <div className="genre-container">
+                  {this.state.singleMovie.genres.map((genre) => (
+                    <p className="movie-genre" key={genre}>
+                      {genre}
+                    </p>
+                  ))}
+                </div>
+              </div>
+            </section>
+          </fragment>
+        )}
         <section className="trailer-container">
           {this.state.singleTrailer && (
             <iframe
               width="600"
               height="400"
               src={`https://www.youtube.com/embed/${this.state.singleTrailer.key}`}
               title="YouTube video player"
-              frameborder="0"
+              frameBorder="0"
               allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
-              allowfullscreen
+              allowFullScreen
             ></iframe>
           )}
         </section>
```

Comment:
We might need to add an error message for users if our fetch calls fail. `{this.state.error && <Error />}`